datasets/dataset_imdb.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+"""
+IMDB dataset.
+inspired from:
+https://keras.io/examples/nlp/text_classification_from_scratch/
+https://builtin.com/data-science/how-build-neural-network-keras
+"""
+
+import tensorflow as tf
+import numpy as np
+from tensorflow import keras
+from keras.utils import to_categorical
+from keras import models
+from keras import layers
+from keras.datasets import imdb
+
+(x_train, y_train), (x_test, y_test) = imdb.load_data(num_words=10000, skip_top=35, start_char=1, oov_char=2, index_from=3)
+
+# Argument num_words => only the num_words first more frequent words are used
+# The others are replaced by 2
+# A Review start by the number 1
+# The longest review is less than 3000 words long
+
+def preprocess_dataset_inputs(data):
+    data_list = []
+    for sample in data:
+        sample += [2] * (nb_words - len(sample)) # 2 is the over_of_vocabulary number
```

Comment:
It would be cleaner if:

- you create a variable for `2` as the oov character, so that you don't have a hardcoded `[2]` and a hardcoded `oov_char=2`
- you define `nb_words` before the definition of this function (for readibility of the code). Maybe rename it to something easier to understand (like `max_nb_words_configured`? something very explicit)

Reply:
This is done